fdb-record-layer-lucene/src/main/java/com/apple/foundationdb/record/lucene/LuceneIndexQueryPlan.java

```diff
@@ -56,6 +57,8 @@
 /**
  * Lucene query plan for including search-related scan parameters.
  */
+@SuppressWarnings("PMD.OverrideBothEqualsAndHashcode")
+@SpotBugsSuppressWarnings("HE_EQUALS_NO_HASHCODE")
```

Comment:
Why did this change result in these warnings being suppressed?

Reply:
I think we had a `hashCode()` before that just called `super.hashCode()` which then caused something checker to barf. Not sure why that only happened when I touched the `computeHashCode...()` but it did. I removed that implementation and another check alerted. I ended up doing it like this.